core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -78,6 +78,7 @@
     private boolean localCompose;
     private boolean pull = true;
     private boolean build = false;
+    private String options = StringUtils.EMPTY;
```

Comment:
Let's use a `Set<String>` to which to add the options.  Or in case order would be important (it probably is not?) a `List`.